### PR TITLE
[one-cmds] Tidy utils.py

### PR DESCRIPTION
**Pull request by jinevening**
This removes duplicate codes in utils.py.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
This is a refactoring work before starting https://github.com/Samsung/ONE/issues/10381.

File: compiler/one-cmds/onelib/utils.py [hunk 1]
```diff
@@ -113,35 +113,24 @@ def parse_cfg(args, driver_name):
...
-                    continue
-                if not is_valid_attr(args, key):
-                    setattr(args, key, config[args.section][key])
-        # if section is not given, section name is same with its driver name
-        else:
-            if not config.has_section(driver_name):
-                raise AssertionError('configuration file must have \'' + driver_name +
-                                     '\' section')
-            secton_to_run = driver_name
-            for key in config[secton_to_run]:
```
Comment (jinevening) [inline compiler/one-cmds/onelib/utils.py:136]: Typo was fixed (secton->section).

**Review by mhs4670go — APPROVED** [review]
LGTM

**Comment by jinevening** [comment]
@seanshpark is on leave today. This PR is just refactoring, so let's land this.

**Pull request merged**